swahili/marshmallow.md

```diff
@@ -1,3 +1,4 @@
 Asubuhi moja, nimeamka ndani ya chumba cha ajabu kikiwa na marshmallow kwenye kuta zake.
 
-(lazima ilikuwa Jumanne.)
\ No newline at end of file
+(lazima ilikuwa Jumanne.)
+[Kuonja marshmallow.](kuonja-marshmallow/kuonja-marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.